parsers/medleydb_parser.py

```diff
@@ -156,13 +172,13 @@ def process(in_dir, out_dir):
     them in the out_dir folder."""
 
     # Collect all trackid's.
-    yaml_files = U.find_with_extension(os.path.join(in_dir, 'Audio'), 'yaml')
-    trackids = [U.filebase(y).replace("_METADATA", "") for y in yaml_files]
+    yaml_files = jams.util.find_with_extension(os.path.join(in_dir, 'Audio'), 'yaml')
```

Comment:
For this, I'd actually pull from within the git repo (there's a folder called 'Metadata'), rather than pulling the Metadata files from the MedleyDB audio distribution.